hub/management/commands/import_rspb_nature_reserves.py

```diff
@@ -0,0 +1,100 @@
+from time import sleep
+
+from django.conf import settings
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaData, DataSet, DataType
+
+from .base_importers import BaseLatLongImportCommand
+
+
+class Command(BaseLatLongImportCommand):
+    help = "Import number of RSPB reserves by constituency"
+
+    data_file = settings.BASE_DIR / "data" / "rspb_reserves_centroids.csv"
+    message = "Importing RSPB reserves data"
+    uses_gss = False
+
+    defaults = {}
+
+    data_sets = {
+        "": {
+            "defaults": defaults,
+        },
+    }
+
+    def add_data_sets(self):
+        data_set, created = DataSet.objects.update_or_create(
+            name="rspb_nature_reserves",
+            defaults={
+                "label": "RSPB Reserves",
+                "data_type": "integer",
+                "category": "place",
```

Comment:
I wonder if this makes more sense as a movement thing, and also have a second data set with the names of the reserves a la national trust?